x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_connect/search_application_api.tsx

```diff
@@ -21,12 +21,9 @@ import {
 } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 
-import { ANALYTICS_PLUGIN } from '../../../../../../common/constants';
-import { COLLECTION_INTEGRATE_PATH } from '../../../../analytics/routes';
 import { CloudDetails, useCloudDetails } from '../../../../shared/cloud_details/cloud_details';
 import { decodeCloudId } from '../../../../shared/decode_cloud_id/decode_cloud_id';
 import { docLinks } from '../../../../shared/doc_links';
-import { generateEncodedPath } from '../../../../shared/encode_path_params';
 import { KibanaLogic } from '../../../../shared/kibana';
```

Comment:
You'll need to remove this and it's usage on line 46 `const { navigateToUrl } = useValues(KibanaLogic);` since it's no longer used.

Reply:
Thanks, done!